### PR TITLE
View Factor Columns - edits

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgFactorLabels.Designer.vb [hunk 1]
```diff
@@ -57,10 +57,10 @@ Partial Class dlgViewFactorLabels
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowPercentage)
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowFrequencies)
         Me.grpSummaryStatistics.Controls.Add(Me.ucrChkShowMissingValues)
-        Me.grpSummaryStatistics.Location = New System.Drawing.Point(316, 199)
+        Me.grpSummaryStatistics.Location = New System.Drawing.Point(170, 199)
         Me.grpSummaryStatistics.Name = "grpSummaryStatistics"
-        Me.grpSummaryStatistics.Size = New System.Drawing.Size(140, 90)
-        Me.grpSummaryStatistics.TabIndex = 5
+        Me.grpSummaryStatistics.Size = New System.Drawing.Size(154, 90)
+        Me.grpSummaryStatistics.TabIndex = 4
```
Comment (lilyclements) [inline instat/dlgFactorLabels.Designer.vb:63]: I moved the Summary Statistics groupbox as I feel it should be next to the Variable Labels groupbox as these are both about the data. Whereas the final groupbox is about layout. This meant changing the tab order.

File: instat/dlgFactorLabels.vb [hunk 2]
```diff
@@ -65,7 +65,7 @@ Public Class dlgViewFactorLabels
         ucrChkShowMissingValues.SetRDefault("FALSE")
 
         ucrChkShowId.SetParameter(New RParameter("show.id", 5))
-        ucrChkShowId.SetText("Show id")
+        ucrChkShowId.SetText("Show ID")
```
Comment (lilyclements) [inline instat/dlgFactorLabels.vb:68]: I think ID should be capital I and D?

**Review by lilyclements — COMMENTED** [review]
Changes outlined below

**Comment by lilyclements** [comment]
@stevenndungu Apologies, don't merge this yet!

**Comment by lilyclements** [comment]
@stevenndungu This is ready to merge

**Comment by lilyclements** [comment]
Recapitilised names. The document I sent earlier this week suggested we don't capitilise these names, and that's really great that you read it. However, after a discussion with Roger we feel we should keep the checkbox names all capitilised

**Comment by stevenndungu** [comment]
Great! This quite OK. I don't objections to the changes. Thanks

**Comment by stevenndungu** [comment]
changes done  and merged...

**Pull request merged**